material-stepper/src/main/java/com/stepstone/stepper/util/AnimationUtil.java

```diff
@@ -0,0 +1,39 @@
+package com.stepstone.stepper.util;
+
+import android.animation.Animator;
+import android.support.annotation.NonNull;
+import android.view.View;
+
+public class AnimationUtil {
```

Comment:
Please add some JavaDoc in this class :) Also `animateView (..)`'s name might be a bit to vague since this method only fades views in and out?